README.md

```diff
@@ -107,6 +107,30 @@ _(Note that in previous versions, the `mysqlserver` environment variable used at
 and the `mysqlserver` environment variable is now no longer supported.)_
 
 
+Instructions to run using minikube and kubectl
+===================================================
+
+You can also run Fineract using containers on a kubernetes cluster.
+
+As Prerequisites, you must have `minikube` and `kubectl` installed on your machine; see
+[Minikube & Kubectl install](https://kubernetes.io/docs/tasks/tools/install-minikube/).
```

Comment:
Just an addon but can update the readme and deployment scripts (if necessary) to show how someone can deploy on other Kubernetes deployment platforms not necessarily mini-kube.
Cause the focus is on Kubernetes, not mini-kube

Reply:
Unfortunately I don't have the resources needed to use an online cluster such GCP/AWS/Azure...
But It will be definitely good to have that in a readme but the configuration should work all thesame. We will be a little stuck at the level of the `minikube start --mount` but that will get fixed up in a follow up PR and also setup bash scripts to automate all the steps used.
But to the best of my knowledge the configuration should work on thesame.

I found an online playground where you can test the configs as per the readme.

Reply:
I'll update the readme with that, but it doesn't seem very ideale to me.